### PR TITLE
[#7]add main view implementation

### PR DESCRIPTION
**Pull request by SwapnilShahGit**
This change includes the polished version of the main view.
Starting electron currently opens up the welcome screen and
the main view. However, after [#24], the multiple windows will
change into just the welcome screen transitioning into the main
view.

### CODE REVIEW

File: css/mainstyle.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+	}
+	.content>div{
+		width: inherit;
+		height: inherit;
+	}
+	.content>iframe{
+		width: inherit;
+		height: inherit;
+	}
+	iframe {
```
Comment (nanuaraj) [inline css/mainstyle.css]: Don't need this any longer.

File: css/mainstyle.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	.content{
+		height: auto;
+		width: auto;
+		padding: 10px;
+	}
+	.content>div{
+		width: inherit;
+		height: inherit;
+	}
+	.content>iframe{
```
Comment (nanuaraj) [inline css/mainstyle.css]: Don't need this

File: css/mainstyle.css [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+		width: 550px;
+		height: 510px;
+    }
+	.frame.maximized{
+		top: 1% !important;
+		left: 1% !important;
+		width: 98% !important;
+		height: 98% !important;
+		-moz-transition: width .25s linear, height .25s linear, top .25s linear, left .25s linear;
+		-webkit-transition: width .25s linear, height .25s linear, top .25s linear, left .25s linear;
```
Comment (nanuaraj) [inline css/mainstyle.css]: Get rid of -(moz, webkit) CSS here

File: css/mainstyle.css [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+		transition: width .25s linear, height .25s linear, top .25s linear, left .25s linear;
+	}
+	.frame.maximized .content{
+		margin: 0 auto !important;
+		width: 95% !important;
+		height: 95% !important;
+		-moz-transition: width .25s linear, height .25s linear, top .25s linear, left .25s linear;
+		-webkit-transition: width .25s linear, height .25s linear, top .25s linear, left .25s linear;
```
Comment (nanuaraj) [inline css/mainstyle.css]: Same as above

File: css/mainstyle.css (hunk 2 above)
Comment (SwapnilShahGit) [reply css/mainstyle.css]: fixed. switched to "webview"

File: css/mainstyle.css (hunk 1 above)
Comment (SwapnilShahGit) [reply css/mainstyle.css]: fixed. switched to "webview"

File: css/mainstyle.css (hunk 4 above)
Comment (SwapnilShahGit) [reply css/mainstyle.css]: fixed.

File: css/mainstyle.css (hunk 3 above)
Comment (SwapnilShahGit) [reply css/mainstyle.css]: fixed.

File: js/mainview.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+$('.frame').mousedown(function(){
+			$(".active").removeClass("active");
```
Comment (hobindar) [inline js/mainview.js]: Indentation is off.

File: js/mainview.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+		$('.frame').not(".maximized").resizable({
+			alsoResize: ".active .content",
+			minWidth: 200,
+			minHeight: 59
+		}).draggable({
+			handle: ".topbar"
+		});
+
+		$('.swatches span').click(function(){
+			var color = $(this).attr("class");
```
Comment (hobindar) [inline js/mainview.js]: Please use `let`.

File: js/mainview.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+$('.frame').mousedown(function(){
```
Comment (hobindar) [inline js/mainview.js]: Maintain spacing before bracket (not just here).

File: main.js [hunk 8]
```diff
@@ -9,11 +9,28 @@ const { ipcMain } = require('electron')
 let mainWindow
 
 function createWindow() {
-    mainWindow = new BrowserWindow({width: 700, height: 600,
-        frame: false,"node-integration": true})
+    welcomeWindow = new BrowserWindow({
+        width: 700, height: 600,
+        frame: false, "node-integration": true
+    })
```
Comment (hobindar) [inline main.js]: All statements should end with a semicolon.

File: css/mainview.css [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+    cursor: default;
+}
+
+.xbtn, .maxbtn {
+    padding: 2px 4px;
+    text-align: center;
+    border-radius: 2px;
+    color: #333;
+    display: inline-block;
+    *display: inline;
```
Comment (hobindar) [inline css/mainview.css]: This appears to be a typo that can be deleted.

@nanuaraj has confirmed this.

File: js/mainview.js (hunk 5 above)
Comment (SwapnilShahGit) [reply js/mainview.js]: fixed.

File: js/mainview.js (hunk 6 above)
Comment (SwapnilShahGit) [reply js/mainview.js]: fixed.

File: js/mainview.js (hunk 7 above)
Comment (SwapnilShahGit) [reply js/mainview.js]: fixed.

File: css/mainview.css (hunk 9 above)
Comment (SwapnilShahGit) [reply css/mainview.css]: fixed.

File: main.js (hunk 8 above)
Comment (SwapnilShahGit) [reply main.js]: fixed.

**Pull request merged**